JSDemos/Demos/PivotGrid/FieldPanel/description.md

```diff
@@ -1 +1,16 @@
-The PivotGrid allows its users to&nbsp;sort and filter data, or&nbsp;manage the collection of&nbsp;fields at&nbsp;runtime. All these operations can be&nbsp;performed using the field panel located at&nbsp;the top of&nbsp;the component. Click the sorting or&nbsp;filtering icon of&nbsp;one of&nbsp;the fields to&nbsp;sort or&nbsp;filter data by&nbsp;this field. Right-click the &laquo;Sales&raquo; field and select an&nbsp;item from the appeared context menu to&nbsp;change the summary type. Drag and drop the fields to&nbsp;rearrange them.
\ No newline at end of file
+The PivotGrid allows its users to&nbsp;sort and filter data, or&nbsp;manage the collection of&nbsp;fields at&nbsp;runtime. All these operations can be&nbsp;performed using the field panel located at&nbsp;the top of&nbsp;the component. Click the sorting or&nbsp;filtering icon of&nbsp;one of&nbsp;the fields to&nbsp;sort or&nbsp;filter data by&nbsp;this field. Right-click the &laquo;Sales&raquo; field and select an&nbsp;item from the appeared context menu to&nbsp;change the summary type. Drag and drop the fields to&nbsp;rearrange them.
```

Comment:
> Click the sorting or filtering icon of one of the fields...

After some time I realised that the filter panel is the top panel and the initial sorting panels for this demo are Date year, Date quarter, Date month. What about mentioning about the structure of field panel?